src/components/App/App.js

```diff
@@ -1,17 +1,10 @@
-import "babel-polyfill";
 import React from "react";
-import {render} from "react-dom";
 import "./app.less";
-import configureStore from "../../store/configureStore";
-import Provider from "react-redux/src/components/Provider";
-import Dashboard from "../Dashboard/Dashboard";
 
-const store = configureStore();
-
-const App = () => (
-    <Provider store={store}>
-        <Dashboard/>
-    </Provider>
+const App = (props) => (
+    <div>
+        {props.children}
+    </div>
 );
 
```

Comment:
import {object} from 'prop-types'; // Do partial importing for performance everywhere you can

Add PropTypes ---> children.object.isRequired